Services/Thoughts.Services/Models/CreatePostDTO.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Thoughts.Services.Models
+{
+    public struct CreatePostDTO
```

Comment:
```suggestion
    public class CreatePostDTO
```
все DTO-структуры надо переделать в DTO-классы. Иначе система привязки модели ASP.NET Core WebAPI не может их создать и выполнить привязку.

Reply:
ОК, понял

Reply:
исправлено